examples/sendingDomains/delete_sendingDomain.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+var key = 'YOURAPIKEY'
+  , SparkPost = require('sparkpost')
+  , client = new SparkPost(key);
+
+client.sendingDomains['delete']('example1.com', function(err, data) {
```

Comment:
We don't need to quote `delete` in Node. If this is failing linting, we should update the jshint to es3:false

I see this in a few places in this PR, I'll just comment this once.